openmrs-client/src/main/java/org/openmrs/mobile/activities/ContactUsActivity.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.mobile.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import com.google.android.material.button.MaterialButton;
+import com.google.android.material.textfield.TextInputEditText;
+import org.openmrs.mobile.R;
+
+
+public class ContactUsActivity extends AppCompatActivity {
+    MaterialButton send;
+    TextInputEditText name, subject, message;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_contact_us);
+        initViewFields();
+        send.setOnClickListener(v -> {
```

Comment:
```suggestion
        send.setOnClickListener(view -> {
```

or `_` if you're not using it